proxy/src/control_plane/client/cplane_proxy_v1.rs

```diff
@@ -29,7 +29,92 @@ use crate::rate_limiter::WakeComputeRateLimiter;
 use crate::types::{EndpointCacheKey, EndpointId};
 use crate::{compute, http, scram};
 
-const X_REQUEST_ID: HeaderName = HeaderName::from_static("x-request-id");
+pub(crate) const X_REQUEST_ID: HeaderName = HeaderName::from_static("x-request-id");
+
+// This client is only used for authentication in ConsoleRedirect proxy
+// and only in case of cancellation requests
+#[derive(Clone)]
+pub struct AuthControlPlaneClient {
```

Comment:
is there any reason why we need to duplicate this and we can't use NeonControlPlaneClient inside of the ConsoleRedirectBackend? We don't need the wake compute rate limiters but it probably would not hurt to create them even if unused.